File: iOS - Using Objective-C/day3/day3Task3_Swipe/day3Task3_Swipe/ViewController.m
//
//  ViewController.m
//  day3Task3_Swipe
//
//  Created by Uef on 19/12/2024.
//

#import "ViewController.h"
#import "SecondViewController.h"
@interface ViewController ()

@end

@implementation ViewController

-(void) goToRight{
    SecondViewController * sVC = [self.storyboard instantiateViewControllerWithIdentifier:@"second"];
    [self presentViewController:sVC animated:YES completion:nil];
}

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    UISwipeGestureRecognizer * stageOne = [[UISwipeGestureRecognizer alloc] initWithTarget:self action:@selector(goToRight)];
    stageOne.direction = UISwipeGestureRecognizerDirectionLeft;
    [self.view addGestureRecognizer:stageOne];
}


@end

File: Design Pattern/Day1/XIB_Demo/XIB_Demo/ViewController.swift
//
//  ViewController.swift
//  XIB_Demo
//
//  Created by Usef on 22/01/2025.
//

import UIKit

class ViewController: UIViewController {
    @IBOutlet weak var containerView: myView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        containerView.myButton.addTarget(self, action: #selector(printHello), for: .touchUpInside)
    }
    @objc func printHello(){
        print("Hello iti!")
        let tVC = self.storyboard?.instantiateViewController(withIdentifier: "table") as! MyTableViewController
        self.navigationController?.pushViewController(tVC, animated: true)
    }


}

